apis/logging/v1/clusterlogging_types.go

```diff
@@ -123,11 +123,16 @@ type ProxySpec struct {
 // This is the struct that will contain information pertinent to Log storage (Elasticsearch)
 type LogStoreSpec struct {
 	// The type of Log Storage to configure
+	//
+	// +kubebuilder:validation:Enum=elasticsearch;lokistack
 	Type LogStoreType `json:"type"`
```

Comment:
I think we need some comment here about how the CLO will not request a lokistack for you like it does for ES.  The assumption here is someone else has already deployed the store.  We need to ensure we are clear that it varies from the previous option